packages/sanity/src/core/preview/types.ts

```diff
@@ -18,27 +18,40 @@ export type Previewable = (
   _internalMeta?: {type?: string}
 }
 
-// @todo: unify with content path from @sanity/types
-export type Path = FieldName[]
+/**
+ * TODO: unify with content path from `@sanity/types`
+ *
+ * @beta
+ */
+export type PreviewPath = FieldName[]
```

Comment:
Here it is; yep something different from Path, and a good idea to differentiate.
Good example of why strict export name checking is gold!